bot/resources/evergreen/trivia_quiz.json

```diff
@@ -604,5 +604,309 @@
+      "answer": "antigravity"
+    },
+    {
+      "id": 510,
+      "question": "Which attribute is the documentation string of a function/method/class stored in (answer should be enclosed in backticks!)?",
+      "answer": "`__doc__`"
+    },
+    {
+      "id": 511,
+      "question": "What is the official name of this operator `:=`, introduced in 3.8?",
+      "answer": "assignment-expression operator"
+    },
+    {
+      "id": 512,
+      "question": "When was Python first released?",
+      "answer": "1991"
+    },
+    {
+      "id": 513,
+      "question": "Where does the name Python come from?",
+      "answer": "Monty Python, Monty Python's Flying Circus"
+    },
+    {
+      "id": 514,
+      "question": "How is infinity represented in Python?",
+      "answer": "float(\"infinity\"), float('infinity'), float(\"inf\"), float('inf')"
+    },
+    {
+      "id": 515,
+      "question": "Which of these characters is valid python outside of string literals in some context? `@`, `$`, `?`",
+      "answer": "@"
+    },
+    {
+      "id": 516,
+      "question": "Which standard library module is designed for making simple parsers for languages like shell, as well as safe quoting of strings for use in a shell?",
+      "answer": "shlex"
+    },
+    {
+      "id": 517,
+      "question": "Which one of these protocols/abstract base classes does the builtin `range` object NOT implement? `Sequence`, `Iterable`, `Generator`",
+      "answer": "Generator"
+    },
+    {
+      "id": 518,
+      "question": "What decorator is used to allow a protocol to be checked at runtime?",
+      "answer": "runtime_checkable, typing.runtime_checkable"
+    },
+    {
+      "id": 519,
+      "question": "Does `numbers.Rational` include the builtin object float (y/n)",
+      "answer": "n, no"
+    },
+    {
+      "id": 520,
+      "question": "What is a package that doesn't have a `__init__` file called?",
+      "answer":"namespace package"
+    },
+    {
+      "id": 521,
+      "question": "What file extension is used by the site module to determine what to do at every start?",
+      "answer": ".pth"
+    },
+    {
+      "id": 522,
+      "question": "What is the garbage collection strategy used by cpython to collect everything but reference cycles?",
+      "answer": "reference counting, refcounting"
+    },
+    {
+      "id": 523,
+      "question": "What dunder method is used by the tuple constructor to optimize converting an iterator to a tuple (answer should be enclosed in backticks!)?",
+      "answer": "`__length_hint__`"
+    },
+    {
+      "id": 524,
+      "question": "Which protocol is used to pass self to methods when accessed on classes?",
+      "answer": "Descriptor"
+    },
+    {
+      "id": 525,
+      "question": "What class was added in a new standard library module for python 3.9?",
+      "answer": "TopologicalSorter"
```

Comment:
there was more than one module added to the stdlib in python 3.9 iirc.

Reply:
hm, i'll check with lak

Reply:
lak never responded- i'll think up a new question and replace it

Reply:
resolved